### ISSUE TITLE
Error in DataTables while suppressing input

### ISSUE THREAD

**Issue by Tazovsky** [aoles/shinyURL#12]
Firstly: 
While supressing input `renderDtOutput_rows_current` using  `.`: `.renderDtOutput_rows_current` package `DataTables` (R package `DT`) returns error. 

Secondly:
`shinyURL` catches every input - including currently rendered rows, e.g.:
`renderDtOutput_rows_current=1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25`
so you can imagine what happens when you render 1000 rows...

Can you add argument to omit specific inputs?

**Comment by aoles** [comment]
Thanks for your feedback! Would you mind including a MWE illustrating the issue?

Cheers,
Andrzej

**Comment by Tazovsky** [comment]
Andrzej,
a powiedz mi proszę czym jest MWE? 

pozdrawiam,
Kamil

**Comment by aoles** [comment]
MWE = minimal working example

czyli najlepiej mozliwie prosta ale kompletna appka ktora bede mogl odpalic u siebie na zasadzie copy-n-paste :)

**Comment by Tazovsky** [comment]
OK, spreparuję przykład jak znajdę chwilę ;)

**Comment by Tazovsky** [comment]
@aoles tutaj MWE: [https://github.com/Tazovsky/shinuURL_MWE/blob/master/app.R](https://github.com/Tazovsky/shinuURL_MWE/blob/master/app.R)

Logi:

> HTTP/1.1 403 Forbidden
Date: Wed, 31 May 2017 10:30:20 GMT
Server: Apache/2.4.18 (Ubuntu)
x-powered-by: Express
Content-Type: text/html; charset=utf-8
X-Remote-User: USERNAME
Keep-Alive: timeout=5, max=65
Connection: Keep-Alive
Transfer-Encoding: chunked

> POST /shiny/user/shinyURL_MWE/session/0b3d326fb1deab7c66e3dfbe2f0f4454/dataobj/.dtOutput?w=&nonce=981c2bad07d514da HTTP/1.1
Host: HOSTNAME
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0
Accept: application/json, text/javascript, */*; q=0.01
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
X-Requested-With: XMLHttpRequest
Referer: https://HOSTNAME/shinyURL_MWE/
Content-Length: 1355
Cookie: user-id=userid|Wed%2C%2021%20Jun%202017%2012%3A47%3A57%20GMT|6mKxUuBZ%2Bb5E1OPa9sH%2BX96qdjPO4jgH4dyNE4wuI8Q%3D; hudson_auto_refresh=true
Authorization: AUTH
Connection: keep-alive

[A tutaj załączam screeny](http://imgur.com/a/zw2qw)

Wydaje mi się, że przez tę `.` może się ten link gryźć z apachem. Dlatego dobrze by było mieć możliwość pominięcia konkretnych inputów.

**Comment by aoles** [comment]
Thank you Kamil for sharing the code. I'm not sure I can reproduce the error, though, at least running the app in RStudio viewer and in Firefox ESR 52.1.2 seems to be fine. Could you maybe clarify how do you run the app and at which stage the error occurs?

**Comment by Tazovsky** [comment]
Yes, it works in RStudio for me too. But not in Firefox. Error occurs just after app start.

**Comment by aoles** [comment]
So which version of FF are you using?
